### PR TITLE
Implement Risk Assessment Questionnaire Feature

### LINKED ISSUE #60: Risk Assessment Questionnaire Feature
As a Financial Advisor, I want to be able to assign risk profiles respective to each client based on the clients' responses to the questionnaire.

### PR DESCRIPTION
**Pull request by Daphne789**
Create a Questionnaire tab that shows the questions for understanding the client's risk profile

Add riskprofile command, which takes in INDEX (index number shown in the displayed person list) and RESULT (client responses to the questions) parameters

Update the existing CSS to color-code the different levels of risk profile shown in person card

This commit is for generating a risk profile instantly based on the client's responses to the questions in the Questionnaire

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/RiskProfileCommand.java [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+            break;
+        case 33:
+        case 34:
+        case 35:
+        case 36:
+        case 37:
+        case 38:
+        case 39:
+        case 40:
+            result = "High";
```
Comment (syamfarh) [inline src/main/java/seedu/address/logic/commands/RiskProfileCommand.java:139]: Why not use if statement instead? this switch statement looks so long

**Review by syamfarh — COMMENTED** [review]
looking good, just a nitpick

File: src/main/java/seedu/address/logic/commands/RiskProfileCommand.java (hunk 1 above)
Comment (Daphne789) [reply src/main/java/seedu/address/logic/commands/RiskProfileCommand.java:139]: Thank you for the suggestion.

**Comment by syamfarh** [comment]
LGTM

**Pull request merged**